src/Context/CachedContextFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Roave\BetterReflection\Context;
```

Comment:
Its the "namespace environment" of the file, i.e. which names are included and which namespace the file resides in.